index.html

```diff
@@ -0,0 +1,367 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+</head>
+<body >
+
+    <style>
+body{
+    
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+nav{
+ 
+ 
+ 
+    display: flex;
+    justify-content: space-around;
+    margin-top: 10px;
+    height: 100%;
+    width: 100%;
+    margin-bottom: 30px;
+}
+#content_search{
+    width:90%;
+    
+    display: flex;
+    justify-content: center;
+}
+input{
+    width: 40%;
+    height: 80%;
+}
+.logo{
+    height: 30%;
+}
+
+#video
+{
+
+height: 100vh;
+    width: 60%;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+   
+
+}
+#rigth_video{
+    
+    height: 100vh;
+    width: 20%;
+    display: flex;
+    flex-direction: column;
+    margin-left: 15px;
+}
+    h3,span
+    
+{
+ font-size:100%;   
+ margin-left: 10px;
+    }
+
+
+iframe{
+
+width: 100%;
+height: 40%;
+
+    
+}
+#title_icons{
+    
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    height: 5%;
+    }
+#iconos{
+    
+    display: flex;
+    width: 8%;
+    justify-content: space-around;
+    
+}article{
+    display: flex;
+    margin-top: 10px;
+}
+#title{
+    
+    width: 100%;
+}
+#icons2{
+    display: flex;
+    justify-content: flex-end;
+    
+
+    
+width: 60%;
+height: 50%;
+}#views{
+    
+    width: 40%;
+height: 50%;
+display: flex;
+justify-content: start;
+
+}
+h3,span{
+    font-size: 100%;
+    margin-left: 2px;
+    align-self: self-end;
+flex-grow: 1;
+}
+
+#subscribe{
+    
+    width: 100%;
+    height: 15%;
+}
+#icon_subscribe{
+margin-top: 10px;
+    
+width: 100%;
+height: 15%;
+display: flex;
+justify-content: space-between;
+    
+}#comment{
+  height: 7%;
+}
+#comment1{
+  width: 100%;
+  height: 60%;
+}
+    #commentSection{
+      margin-top: 15px;
+
+display: flex;
+
+    }
+    #picture{
+      width: 6%
+    }
+    #picture_comment{
+      width: 90%
+    }
+
+
+   </style>
```

Comment:
- Good job, is better, and suggested use styles in a new file , this practice is the best way to keep your code clean,organized and smaller size. 👍
- Try to use linters to check anomalies, wrong style, syntax. (details about this at the main comment). 😀